cmd/errtrace/main.go

```diff
@@ -93,6 +93,7 @@ func (p *mainParams) Parse(w io.Writer, args []string) error {
 	if len(p.Files) == 0 {
 		// Read file from stdin when there's no args, similar to gofmt.
 		p.Files = []string{"-"}
+		fmt.Fprintln(w, "reading from stdin; use '-' to hide this message and '-h' for help")
```

Comment:
couple of optional ideas,

 - should we include the usage here?
 - the most common case where this is useful is when the user runs the command directly, in which case stdin will be a TTY. do you think it's worth checking if we have a TTY before printing this?

Reply:
rather than trying to add platform specific TTY-detection, I've changed this so that it only prints the message if there's no input on implicit stdin for 200 milliseconds.

For most users, that'll be near instantaneous, but it won't print for stdin pipes.